db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150422211011) do
+ActiveRecord::Schema.define(version: 20150502222447) do
```

Comment:
[rubocop] Separate every 3 digits in the integer portion of a number with underscores(_).